tests/test_data/test_dataset.py

```diff
@@ -26,6 +27,40 @@ def test_classes():
         get_classes('unsupported')
 
 
+def test_classes_file_path():
+    train_pipeline = [dict(type='LoadImageFromFile')]
+    classes_path = '../data/classes.txt'
```

Comment:
Please use `tempfile.NamedTemporaryFile()`.
You can refer to 
https://github.com/open-mmlab/mmsegmentation/blob/186a1fc88e31f294adbabb677608240aa2221d2a/tests/test_models/test_losses/test_dice_loss.py#L28

Reply:
ok, I will modify the test sample.

Reply:
https://github.com/open-mmlab/mmcv/blob/c47c9196d067a0900b7b8987a8e82768edab2fff/tests/test_fileio.py